prisma/schema.prisma

```diff
@@ -86,14 +86,13 @@ model Application {
 }
 
 model Employee {
-  id            Int       @id @default(autoincrement())
-  firstName     String    @map("first_name") @db.VarChar(255)
-  lastName      String    @map("last_name") @db.VarChar(255)
-  email         String    @unique @db.VarChar(255)
-  emailVerified DateTime? @map("email_verified") @db.Timestamptz(6)
-  role          Role
-  createdAt     DateTime  @default(now()) @map("created_at") @db.Timestamptz(6)
-  updatedAt     DateTime  @default(now()) @updatedAt @map("updated_at") @db.Timestamptz(6)
+  id        Int      @id @default(autoincrement())
+  firstName String   @map("first_name") @db.VarChar(255)
+  lastName  String   @map("last_name") @db.VarChar(255)
+  email     String   @unique @db.VarChar(255)
```

Comment:
Don't know why `emailVerified` got deleted here. This is very likely the reason for the build failing.